packages/belos/src/BelosMinresIter.hpp

```diff
@@ -403,12 +403,13 @@ class MinresIter : virtual public MinresIteration<ScalarType,MV,OP> {
     // Create convenience variables for zero, one.
     const ScalarType one = SCT::one();
     const MagnitudeType zero = SMT::zero();
+    const MagnitudeType m_zero = SMT::zero();
```

Comment:
How is the m_zero here different from zero?

Reply:
It's a zero for the magnitude type.  It's easier to identify the difference between the scalar type zero and magnitude type zero by using the "m_" prefix.  It is not used consistently throughout the code, but it helps with clarity.

Reply:
Did you notice that the 'zero' above here is also a magnitude type?